docs/DeveloperGuide.md

```diff
@@ -420,15 +421,24 @@ The TransactionList class is responsible for managing a list of transactions in
 
 - *deleteMember*: Deletes a member from all transactions in the list.
 
+- *filterTransactionsEqualToDateTime*: Lists all transactions with dateTime equal to the input String represented dateTime
+
```

Comment:
Remove these newlines. It affects the formatting of thee page

Reply:
Resolved!